src/test/java/de/tum/in/www1/artemis/StudentQuestionIntegrationTest.java

```diff
@@ -214,4 +214,41 @@ public void deleteStudentQuestions_asStudent() throws Exception {
         request.delete("/api/student-questions/" + studentQuestion1_student2.getId(), HttpStatus.FORBIDDEN);
         assertThat(studentQuestionRepository.count()).isEqualTo(1);
     }
+
+    @Test
+    @WithMockUser(username = "instructor1", roles = "INSTRUCTOR")
+    public void editStudentQuestionVotes_asInstructor() throws Exception {
+        StudentQuestion studentQuestion = database.createCourseWithExerciseAndStudentQuestions().get(0);
+
+        studentQuestion.setVotes(42);
+
+        StudentQuestion updatedStudentQuestion = request.putWithResponseBody("/api/student-questions/" + studentQuestion.getId() + "/votes", 0, StudentQuestion.class,
+                HttpStatus.OK);
+        assertThat(updatedStudentQuestion.getVotes().equals(42));
+    }
+
+    @Test
+    @WithMockUser(username = "tutor1", roles = "TA")
+    public void editStudentQuestionVotes_asTA() throws Exception {
+        StudentQuestion studentQuestion = database.createCourseWithExerciseAndStudentQuestions().get(0);
+
+        studentQuestion.setVotes(42);
+
+        StudentQuestion updatedStudentQuestion = request.putWithResponseBody("/api/student-questions/" + studentQuestion.getId() + "/votes", 0, StudentQuestion.class,
+                HttpStatus.OK);
+        assertThat(updatedStudentQuestion.getVotes().equals(42));
+    }
+
+    @Test
+    @WithMockUser(username = "student1", roles = "USER")
+    public void editStudentQuestionVotes_asStudent() throws Exception {
+        List<StudentQuestion> questions = database.createCourseWithExerciseAndStudentQuestions();
+        StudentQuestion studentQuestion = questions.get(0);
+
+        studentQuestion.setVotes(42);
+
+        StudentQuestion updatedStudentQuestion = request.putWithResponseBody("/api/student-questions/" + studentQuestion.getId() + "/votes", 0, StudentQuestion.class,
+                HttpStatus.OK);
+        assertThat(updatedStudentQuestion.getVotes().equals(42));
+    }
```

Comment:
I believe some test cases where the value actually gets changed would be quite useful.

Reply:
These were just placeholders, now there are tests that actually test something with https://github.com/ls1intum/Artemis/pull/1478/commits/45b799f736ae7a4a3ae31813144fb489be783d9b

Reply:
Yeah, thats how I like them test cases :D